src/select/__tests__/select.test.js

```diff
@@ -302,28 +308,29 @@ describe('Stateless select', function() {
         .find(StyledInput)
         .first()
         .find('input');
+
       input.simulate('change', {target: {value: 'a'}});
-      onTextInputChangePromise.then(() =>
-        expect(wrapper.find(StyledOption)).toHaveLength(3),
-      );
+      await sleep(20);
```

Comment:
currently tests were asynced, but only by 0 to 1 tick of processor and only because JS released thread while  Promise works in the [next tick](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises#Timing). But now they are slower by 20ms, multiplied by their quantity.
Tests should be as fast as possible to pass [quickly for many reasons](https://github.com/ghsukumar/SFDC_Best_Practices/wiki/F.I.R.S.T-Principles-of-Unit-Testing)

Reply:
Yeah I agree–it didn't feel great adding these sleep statements here but I was struggling to figure out a way to do this that wouldn't be flaky. `await onTextInputChangePromise` didn't work (same as `.then` logic that existed before)

Will try to remove this sleep or lower the number of milliseconds.

Reply:
Got it down to `sleep(1)`